modules/python/src/custom_nodes/google_genmedia/constants.py

```diff
@@ -36,6 +36,7 @@
 GEMINI_25_FLASH_IMAGE_USER_AGENT = (
     "cloud-solutions/comfyui-gemini-25-flash-image-custom-node-v1"
 )
+GOOGLE_GOOGLE_TTS_USER_AGENT = "cloud-solutions/comfyui-tts-custom-node-v1"
```

Comment:
recommendation is to use the name of the model(Chirp) like other nodes instead of tts